### PR TITLE
Grid Layout added in Home Fragment along with ScrollView

### PR DESCRIPTION
**Pull request by Joaquin144**
Constraint --> ScrollView --> Constraint --> TV and RV

### CODE REVIEW

File: app/src/main/java/com/ersubhadip/hackathon/fragment_home.java [hunk 1]
```diff
@@ -4,43 +4,81 @@
...
+//         manager.setOrientation(RecyclerView.HORIZONTAL);
+//         homeRv.setLayoutManager(manager);
+//         homeRv.setAdapter(adapter);
+//         adapter.notifyDataSetChanged();
+
+         //Humko Grid banana hai
+         GridLayoutManager gridLayoutManager = new GridLayoutManager(getContext(),2,GridLayoutManager.VERTICAL,false);
+         homeRv.setLayoutManager(gridLayoutManager);
+         homeRv.setAdapter(adapter);
+         homeRv.hasFixedSize();//pata nahi kya karti hai ye line
```
Comment (RahulSoni0) [inline app/src/main/java/com/ersubhadip/hackathon/fragment_home.java:80]: lol , pta nhi phỉ bhi 🤷🏻

**Comment by Subhadiptech** [comment]
Will Accept after Review @RahulSoni0 .
Review Pending

**Comment by Subhadiptech** [comment]
Ok Checked . @RahulSoni0 merge to master

@Joaquin144  avoid using hindi comments😅🤣🤣

**Comment by Subhadiptech** [comment]
Thanks @Joaquin144

**Pull request closed without merging**